db/connection_testing/connection_testing.go

```diff
@@ -728,3 +733,108 @@ func testConnector_Regression_Returning(t *testing.T, newDB NewDB) {
 		t.FailNow()
 	}
 }
+
+func testConnector_ExecResult(t *testing.T, newDB NewDB) {
+	db := newDB(t)
+
+	rand.Seed(time.Now().UnixNano())
+	tempID1 := rand.Intn(11000) + 10
+	tempID2 := rand.Intn(11000) + 10
+	tempID3 := rand.Intn(11000) + 10
+	initialDesc1 := uuid.NewV4().String()
+	initialDesc2And3 := uuid.NewV4().String()
+
+	insertQuery := chain.NewExpresionChain(db)
+	_, err := insertQuery.InsertMulti(
+		map[string][]interface{}{
+			"id":          []interface{}{tempID1, tempID2, tempID3},
+			"description": []interface{}{initialDesc1, initialDesc2And3, initialDesc2And3},
+		})
+	insertQuery.Table("justforfun")
+	if err != nil {
+		t.Logf("failed to generate insertQuery: %v", err)
+		t.FailNow()
+	}
+	rowsAffected, err := insertQuery.ExecResult()
+	if err != nil {
+		t.Logf("failed to insert: %v", err)
```

Comment:
`t.Fatalf("failed to insert: %v", err)` would be one line shorter.

Reply:
Agreed, but favoring local conventions of file (cargo culting?)